File: go/update.go
package main

import (
	"context"
	"fmt"
	"log"

	dub "github.com/dubinc/dub-go"
	"github.com/dubinc/dub-go/models/operations"
)

func updateLink(d *dub.Dub) {
	var request *operations.UpdateLinkRequestBody = &operations.UpdateLinkRequestBody{
		URL: "https://google.us",
	}

	ctx := context.Background()
	res, err := d.Links.Update(ctx, "LINK_ID", request)
	if err != nil {
		log.Fatal(err)
	}
	if res.LinkSchema != nil {
		fmt.Println(res.LinkSchema.ShortLink)
	}
}

File: ruby/old/update.rb
require 'dub'

s = ::OpenApiSDK::Dub.new

s.config_security(
  ::OpenApiSDK::Shared::Security.new(
    token: ENV['DUB_API_KEY'],
  )
)

req = ::OpenApiSDK::Operations::UpdateLinkRequest.new(
  link_id: "cly2p8onm000cym8200nfay7l",
  request_body: ::OpenApiSDK::Operations::UpdateLinkRequestBody.new(
    url: "https://google.us",
  ),
)
    
res = s.links.update(req)

puts res.raw_response.body